src/contracts/strategies/StrategyBaseTVLLimits.sol

```diff
@@ -79,8 +79,8 @@ contract StrategyBaseTVLLimits is StrategyBase {
      * @param amount The amount of `token` being deposited
      */
     function _beforeDeposit(IERC20 token, uint256 amount) internal virtual override {
-        require(amount <= maxPerDeposit, "StrategyBaseTVLLimits: max per deposit exceeded");
-        require(_tokenBalance() <= maxTotalDeposits, "StrategyBaseTVLLimits: max deposits exceeded");
+        require(amount <= maxPerDeposit, MaxPerDepositExceedsMax());
+        require(_tokenBalance() <= maxTotalDeposits, MaxPerDepositExceedsMax());
```

Comment:
This should be a separate error `MaxDepositsExceeded()` since this isn't PER deposit like above